src/packages/engine-core/src/maskQuery.ts

```diff
@@ -0,0 +1,10 @@
+export function maskQuery(query?: string): string {
+  if (!query) {
+    return ''
+  }
+  return query
+    .replace(/".*"/g, '"X"')
+    .replace(/[\s:\[]([+-]?([0-9]*[.])?[0-9]+)/g, (substr) => {
+      return `${substr[0]}5`
+    })
```

Comment:
I guess the first replace is for turning all string values to "X".
The second one would be nice with a comment of what it does I think.

Reply:
It replaces numbers with `5` - just added a comment